robottelo.properties.sample

```diff
@@ -2,6 +2,10 @@
 [server]
 # Server hostname
 hostname=
+# Additional satellite hostnames for xdist workers
+# gw0=
+# gw1=
+# gw2=
```

Comment:
Is it capped to only 3 additional workers , or it can be any numbers of workers ?

If yes, Can we atleast define upto `gw7` here ? OR inform the availability of any number of workers in the comment.

Reply:
Its unlimited in the behavior, you could have 100 xdist workers if you've got 100 instances to list here. But the machine running pytest better have a _lot_ of cores and memory :D